api_app/api.py

```diff
@@ -512,6 +463,54 @@ def get_serializer_class(self, *args, **kwargs):
         except (KeyError, AttributeError):
             return super(JobViewSet, self).get_serializer_class()
 
+    @action(detail=True, methods=["patch"])
```

Comment:
Shouldn't it be `detail=False`? We do not want this action to be available to collections /lists.

Reply:
It should be `True.